### DISCUSSION TITLE
Frequency List Edit..??

### DISCUSSION THREAD

**Discussion by Wirezfree** [N0BOY/FT8CN#81] [Q&A]
Hello,

Is it possible to Edit the Frequency List
I want to reduce the entries to match my QRPLabs QDX TX/RX, and I only want FT8 Frequencies.

Many Thanks... Dave

**Comment by bg7yoz** [comment]
hi，@Wirezfree 

非常好的建议！但是，在手机屏幕上修改频率列表并不是一件容易的事---受限于屏幕的尺寸。如果有可能，我会把修改频率列表的操作放在后台（Web）。

73！

bg7yoz

**Reply by Wirezfree** [reply]
Hello,

Many Thanks for quick reply...

I don't think this is limited by the screen.??
When the Operating Frequency list is opened..
Tap and Hold on the Frequency, Option appears
"Delete Frequency.?:"
 [YES] / [NO]
Tap [YES]
"Are You Sure.?":
[YES] / [NO]
If you Tap [YES]
The Frequency entry is then Deleted from Operating Frequency List

Thanks & Regards
Dave

**Comment by 7K1BIB** [comment]
I also want to edit the frequency list in order to add 430.510MHz, FT8 frequency in Japan.  How can I do that?

**Comment by danleetw** [comment]
Please try this one.
https://github.com/danleetw/FT8TW

**Reply by 7K1BIB** [reply]
Thank you for your information.  That is what I needed.
